packages/page-staking/src/util.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017-2025 @polkadot/app-staking authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Option } from '@polkadot/types';
+import type { Codec } from '@polkadot/types/types';
+import type { ChainAbftScore } from './types.js';
+
+export function decodeChainAbftScore (chainAbftScore: Codec | undefined, session: number) {
```

Comment:
it seems to me that `session` here serves no purpose. it's pass in and passed out. Could you remove it?